src/app/components/turnos/gestor-agendas/pipes/botonesAgenda.pipe.ts

```diff
@@ -0,0 +1,92 @@
+import { Auth } from '@andes/auth';
+import { Pipe, PipeTransform } from '@angular/core';
+import { IAgenda } from '../../../../interfaces/turnos/IAgenda';
+
+@Pipe({
+    name: 'botonesAgenda'
+})
+export class BotonesAgendaPipe implements PipeTransform {
+    constructor(private auth: Auth) { }
+    transform(agenda: IAgenda): any {
+        const botones = {
+            editar: false,
+            suspender: false,
+            reasignar: false,
+            clonar: false,
+            sobreturno: false,
+            revision: false,
+            cargaMasiva: false
+        };
+
+        const puedeEditar = this.auth.check('turnos:agenda:puedeEditar:');
+        const puedeClonar = this.auth.check('turnos:agenda:puedeClonar:');
+        const puedeRevisar = this.auth.check('turnos:agenda:puedeRevision:');
+        const puedeSuspender = this.auth.check('turnos:agenda:puedeSuspender:');
+        const puedeDarSobreturno = this.auth.check('turnos:agenda:puedeDarSobreturno:');
+        const puedeReasignar = this.auth.check('turnos:agenda:puedeReasignar:');
+        const puedeCargar = this.auth.check('turnos:agenda:asignacionMasiva:');
```

Comment:
los dos puntos finales están de mas
`this.auth.check('turnos:agenda:puedeEditar');`

Reply:
Corregido